Samples/BruTile.Demo.Uno/BruTile.Demo.Uno.Shared/MapControl.cs

```diff
@@ -0,0 +1,192 @@
+﻿using System;
```

Comment:
This is so much code. This needs to be maintained. Could the Mapsui package itself be used to present these samples? (needs to be released first).

Reply:
Yes I think this would be better, and making it a simpler example. I will slim down this pull request, for that.

Reply:
Also not sure if we need a sample. For your development it was good to see wasm now actually worked, but now it does do we still need it? Mapsui already has a dependency on Uno, so you could show how it works in there. Is there another benefit of the sample that could not be shown in the other samples?

Reply:
Then I remove the Uno Sample, because it works better in Mapsui (probably not tested yet)